hera_mc/geo_handling.py

```diff
@@ -117,8 +118,7 @@ def cofa(self):
     def get_station_types(self, add_stations=True):
```

Comment:
As in a recent PR, you might consider renaming this to `ensure_station_types()`.